### PR TITLE
swashbuckle.aspnetcore being removed in dotnet 9

### LINKED ISSUE #33296: swashbuckle.aspnetcore being removed in dotnet 9
### Description

According to https://github.com/dotnet/aspnetcore/issues/54599 the swashbuckle.aspnetcore package will be removed in dotnet 9. It'd be helpful if this documentation mentioned this. Thanks!

[Enter feedback here]

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-8.0&tabs=visual-studio

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/getting-started-with-swashbuckle.md

### Document ID

59222460-a971-e76e-4fe9-23d4d7d768ee

### Article author

@zuckerthoben

---
[Associated WorkItem - 295452](https://dev.azure.com/msft-skilling/Content/_workitems/edit/295452)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #33296

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/getting-started-with-swashbuckle.md](https://github.com/dotnet/AspNetCore.Docs/blob/e32115c991e03c61341bacc7ae8d470c3921a623/aspnetcore/tutorials/getting-started-with-swashbuckle.md) | [Get started with Swashbuckle and ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?branch=pr-en-us-33545) |

### CODE REVIEW

File: aspnetcore/tutorials/getting-started-with-swashbuckle/includes/getting-started-with-swashbuckle8.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 
 :::moniker range=">= aspnetcore-6.0 < aspnetcore-9.0"
 
+> [!NOTE]
+> Swashbuckle is not available in .NET 9 and later. For an alternative, see <xref:fundamentals/openapi/overview>.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/getting-started-with-swashbuckle/includes/getting-started-with-swashbuckle8.md:5]:
```suggestion
> Swashbuckle is not available in .NET 9 and later. For an alternative, see <xref:fundamentals/openapi/overview?view=aspnetcore-9.0&preserve-view=true>.
```

File: aspnetcore/tutorials/getting-started-with-swashbuckle/includes/getting-started-with-swashbuckle8.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 
 :::moniker range=">= aspnetcore-6.0 < aspnetcore-9.0"
 
+> [!NOTE]
+> Swashbuckle is not available in .NET 9 and later. For an alternative, see <xref:fundamentals/openapi/overview?view=aspnetcore-9.0&preserve-view=true>.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/getting-started-with-swashbuckle/includes/getting-started-with-swashbuckle8.md:5]: @tdykstra  see e32115c991e03c61341bacc7ae8d470c3921a623 which I think is too tricky.

**Review by tdykstra — APPROVED** [review]
Looks good, but we'll have a better target for the link when my PR (to be created later today) that creates the "using the openapi document" article merges .

**Comment by Rick-Anderson** [comment]
> Looks good, but we'll have a better target for the link when my PR (to be created later today) that creates the "using the openapi document" article merges .

You'll have to remember that because I won't.

**Pull request merged**